packages/api/src/external/carequality/document/process-outbound-document-query-resps.ts

```diff
@@ -77,14 +77,26 @@ export async function processOutboundDocumentQueryResps({
     const replaceDqUrlWithDrUrl = async (
       outboundDocumentQueryResp: OutboundDocumentQueryResp
     ): Promise<void> => {
-      const gateway = await getCQDirectoryEntryOrFail(
-        outboundDocumentQueryResp.gateway.homeCommunityId
-      );
+      const gateway = await getCQDirectoryEntry(outboundDocumentQueryResp.gateway.homeCommunityId);
 
-      if (!gateway.urlDR) {
+      if (!gateway) {
+        const msg = `Gateway not found`;
```

Comment:
Maybe add the "leg"/operation being executed as well? I see we have the context which is good, just for when we see the notif before opening Sentry we kinda know what it is related to.

Here it would be "Doc Query", right?